components/prospects/CreateProspectNote.tsx

```diff
@@ -0,0 +1,81 @@
+import { useState } from 'react'
```

Comment:
I would put this file in a separate folder called notes. In the future we might want to have notes for other things as well

Reply:
Are you saying that I should change the location of the file to: 'components/note/CreateProspectNote.tsx'?

Reply:
note -> notes, but yes